lib/chef/resource/chef_client_scheduled_task.rb

```diff
@@ -122,7 +122,8 @@ class ChefClientScheduledTask < Chef::Resource
 
       property :config_directory, String,
         description: "The path of the config directory.",
-        default: ChefConfig::Config.etc_chef_dir
+        default: ChefConfig::Config.etc_chef_dir,
+        default_description: "`C:\\chef\\`"
```

Comment:
Could you make use of `ChefConfig::Config.c_chef_dir` instead of static value?

Reply:
yep. thanks!